### PR TITLE
added polish_00.txt

### PR DESCRIPTION
**Pull request by PiotrKowalski**

### CODE REVIEW

File: test/queries/polish_00.txt [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+Young Thug
+Mozilla
+The International 5
+wykop
+RefugeeCrisis
+Kendrick Lamar
+Angela Merkel
+Donald Trump
+ataki na paryz
+13.12
```
Comment (sudheesh001) [inline test/queries/polish_00.txt]: @PiotrKowalski What is `13.12` ? Is there a reason behind this ? I just made a sample twitter search and ended up with really wierd results. Can you explain what this is ?

**Comment by PiotrKowalski** [comment]
I thought about attacks in Paris. I confused my dates :P I can repair it. And I copied a lot of things from english_00 because I did 200 phrases in polish and file disappeared so I had to new one

**Comment by sudheesh001** [comment]
@PiotrKowalski There are 199 here, Looks like you need to add one more and also please correct the date if you remember or replace it with a new search query.

**Comment by PiotrKowalski** [comment]
Sorry, I tried squash 2 commits but I created new 4. I won't try again, sorry :D And I did what I had to.

**Comment by sudheesh001** [comment]
@PiotrKowalski I am extremely sorry, I tried to rebase this and it's caused way too many problems and erases a few other commits because when you sent this PR the master had moved ahead in a few commits. Unfortunately git is unable to figure out merges as a part of soft rebase. 

I'll be closing this PR, Could you please redo this, but before you do, since you're doing it from your master, I suggest you please reset master to
`git reset --hard 47fe33e0e6f12090dbb378070b66c35357313077`
Then 
`git pull https://github.com/loklak/loklak_server master`
then
`git checkout -b Polish200`
Followed by
`git add polish_00.txt`
`git push origin Polish200` and then make a pull request.

:smile:

**Comment by Orbiter** [comment]
hey, I liked the word list, lets merge it!

**Comment by sudheesh001** [comment]
@Orbiter merged in #162

**Pull request closed without merging**